client/ayon_houdini/plugins/create/create_FBX.py

```diff
@@ -175,3 +190,16 @@ def get_selection(self):
             )
 
         return selection
+
+
+class createStaticMesh(CreateFBX):
+    """Static Meshes as FBX. """
+
+    identifier = "io.openpype.creators.houdini.staticmesh.fbx"
+    label = "staticMesh (FBX)A"
+    product_type = "staticMesh"
+    icon = "cube"
+    default_variants = ["Main"]
+
+    # Default render target
+    render_target = "local"
```

Comment:
```suggestion

```
Anything you do not override does not need to be redeclared on the child class - so let's remove these.